lib/fastlane/plugin/wpmreleasetoolkit/actions/common/update_pull_requests_milestone_action.rb

```diff
@@ -0,0 +1,104 @@
+require 'fastlane/action'
+require_relative '../../helper/github_helper'
+
+module Fastlane
+  module Actions
+    class UpdatePullRequestsMilestoneAction < Action
+      def self.run(params)
+        repository = params[:repository]
+
+        github_helper = Fastlane::Helper::GithubHelper.new(github_token: params[:github_token])
+
+        pr_number = params[:pr_number]
+        from_milestone = params[:from_milestone]
+        to_milestone = params[:to_milestone]
+        pr_comment = params[:pr_comment]
+
+        target_milestone = nil
+        unless to_milestone.nil?
+          target_milestone = github_helper.get_milestone(repository, to_milestone)
+          UI.user_error!("Unable to find target milestone matching version #{to_milestone}") if target_milestone.nil?
+        end
+
+        prs_nums = if pr_number
+                     [pr_number]
+                   elsif from_milestone
+                     # get the milestone object based on title starting text
+                     m = github_helper.get_milestone(repository, from_milestone)
+                     UI.user_error!("Unable to find source milestone matching version #{from_milestone}") if m.nil?
+
+                     # get all open PRs in that milestone
+                     github_helper.get_prs_for_milestone(repository: repository, milestone: m).map(&:number)
+                   else
+                     UI.user_error!('One of `pr_number` or `from_milestone` must be provided, to indicate which PR(s) to update')
```

Comment:
I believe this comma isn't needed?

```suggestion
                     UI.user_error!('One of `pr_number` or `from_milestone` must be provided to indicate which PR(s) to update')
```